### DISCUSSION TITLE
how to pass arguments with command line

### DISCUSSION THREAD

**Discussion by drdedou** [seleniumbase/SeleniumBase#2534] [Q&A] [answered]
ERROR: file or directory not found: testname

**Comment by mdmintz** [comment, marked as answer]
Learn all about passing `pytest` command-line options here:
[SeleniumBase/help_docs/customizing_test_runs.md](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/customizing_test_runs.md)

**Comment by drdedou** [comment]
thank you
